### PR TITLE
[MIG] project_status: Migration to 15.0

### PR DESCRIPTION
**Pull request by igallart**

### CODE REVIEW

File: project_status/models/project.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+from odoo import SUPERUSER_ID, api, fields, models
+
+
+class Project(models.Model):
+    _inherit = "project.project"
+
+    @api.model
+    def _read_group_status_ids(self, statuses, domain, order):
+        statuse_ids = statuses._search([], order=order, access_rights_uid=SUPERUSER_ID)
```
Comment (ajaniszewska-dev) [inline project_status/models/project.py:9]:
```suggestion
        status_ids = statuses._search([], order=order, access_rights_uid=SUPERUSER_ID)
```

i'd rename that one.

**Review by ajaniszewska-dev — APPROVED** [review]

File: project_status/models/project.py (hunk 1 above)
Comment (igallart) [reply project_status/models/project.py:9]: Fine with me.

**Review by rousseldenis — APPROVED** [review]
Code review

File: project_status/models/project.py (hunk 1 above)
Comment (leemannd) [reply project_status/models/project.py:9]: Hello @igallart do you want to include this before it's merged?

Comment (igallart) [reply project_status/models/project.py:9]: Hello @leemannd , I don't have this module very fresh, I just tried to migrate it to make it functional, so I don't know how it works 100%. If you think that code is not necessary, you are free to remove it, I can't help you much more.

**Review by antoniocanovas — APPROVED** [review]
Functional review OK

**Pull request merged**